### DISCUSSION TITLE
New incoming ticket contains correct time&date headers–Cerb displays the ticket as being 8 months old

### DISCUSSION THREAD

**Discussion by j-at-se** [cerb/cerb-release#78] [Q&A] [answered]
Hello and thank you in advance for your help.

Our workers reported an issue with a new message from an old ticket showing up as being 8 months old.

![Screenshot at Sep 05 20-48-16](https://github.com/user-attachments/assets/dc0c3329-4855-442a-8ff5-796f8545041d)

This seems to have triggered another issue but I believe this is the root cause.

Is this a bug or something that could happen for known reasons?

**Comment by Hildy** [comment, marked as answer]
We use the Date as specified in the Date header of the email, because there's no guarantee that we'll be processing that email when it was sent.  If you look further down in those headers, you'll almost certainly find that the Date header on the email matches the one displayed in Cerb.

**Reply by j-at-se** [reply]
Thank you! You're absolutely correct.
